src/anshuKJun20/NumberArrayOperation.java

```diff
@@ -0,0 +1,38 @@
+package anshuKJun20;
+
+import java.util.Scanner;
+
+public class NumberArrayOperation {
+	static Scanner scanner = new Scanner(System.in);
+	
+public static void main(String[] args){
+	
```

Comment:
Find minNumber method is missing.

Reply:
![ArrayOperation](https://user-images.githubusercontent.com/66829451/85304967-03e5b280-b4ca-11ea-9c91-a8699ef89efd.PNG)
Changes Done